app/models/vendor.rb

```diff
@@ -0,0 +1,10 @@
+class Vendor < ApplicationRecord 
+	validates :name, :description, :contact_name, :contact_phone, presence: true
+	validates :credit_accepted, inclusion: { in: [true, false] }
```

Comment:
Ah yes, I ran into this issue as well and was why I ended up needing to build a custom validation for this attribute of Vendor. I believe from my understanding, the value of false is read as 'nil' somewhere along the line, so the validation can get thrown off when trying to handle the error of creating or updating a Vendor record and break.